power_grid_model_c/power_grid_model/include/power_grid_model/calculation_parameters.hpp

```diff
@@ -148,6 +154,7 @@ struct MathModelTopology {
     DenseGroupedIdxVector power_sensors_per_branch_from;
     DenseGroupedIdxVector power_sensors_per_branch_to;
     DenseGroupedIdxVector power_sensors_per_bus;
+    DenseGroupedIdxVector transformer_tap_regulators_per_branch;
```

Comment:
`tap_regulator` is only present in `transformer` objects, hence redundant:
```suggestion
    DenseGroupedIdxVector tap_regulators_per_branch;
```

Reply:
whoops should be out of scope of this PR